Frameworks/include/CoreGraphics/D2DWrapper.h

```diff
@@ -0,0 +1,28 @@
+//******************************************************************************
+//
+// Copyright (c) Microsoft. All rights reserved.
+//
+// This code is licensed under the MIT License (MIT).
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+// THE SOFTWARE.
+//
+//******************************************************************************
+
+#pragma once
+
+#include <COMIncludes.h>
+#import <wrl/implements.h>
+#import <D2d1.h>
+#include <COMIncludes_End.h>
+
+using namespace std;
+using namespace Microsoft::WRL;
+
+static ComPtr<ID2D1Factory> _CreateD2DFactoryInstance(D2D1_FACTORY_TYPE type);
+static ComPtr<ID2D1Factory> _GetD2DFactoryInstance();
```

Comment:
>static [](start = 0, length = 6)

static here doesn't make much sense.  use static on methods if they are only supposed to be used in one file, in this case, it will be used across different files, so this doesnt apply.